### ISSUE TITLE
About the 'center_origin_cls_labels'

### ISSUE THREAD

**Issue by dk-liang** [yifanzhang713/IA-SSD#5]
Thanks for your interesting work.
I now try to train on my own dataset. But in some cases, it meets a bug, as shown below:

```
  File "../pcdet/models/detectors/IASSD.py", line 13, in forward
    loss, tb_dict, disp_dict = self.get_training_loss()
  File "../pcdet/models/detectors/IASSD.py", line 24, in get_training_loss
    loss_point, tb_dict = self.point_head.get_loss()
  File "../pcdet/models/dense_heads/IASSD_head.py", line 413, in get_loss
    center_loss_reg, tb_dict_3 = self.get_contextual_vote_loss()
  File "../pcdet/models/dense_heads/IASSD_head.py", line 470, in get_contextual_vote_loss
    center_origin_loss_box = torch.cat(center_origin_loss_box, dim=-1).mean()
NotImplementedError: There were no tensor arguments to this function (e.g., you passed an empty list of Tensors), but no fallback function is registered for schema aten::_cat.  This usually means that this function requires a non-empty list of Tensors, or that you (the operator writer) forgot to register a fallback function.  Available functions are [CPU, CUDA, QuantizedCPU, BackendSelect, Python, Named, Conjugate, Negative, ADInplaceOrView, AutogradOther, AutogradCPU, AutogradCUDA, AutogradXLA, AutogradLazy, AutogradXPU, AutogradMLC, AutogradHPU, AutogradNestedTensor, AutogradPrivateUse1, AutogradPrivateUse2, AutogradPrivateUse3, Tracer, UNKNOWN_TENSOR_TYPE_ID, Autocast, Batched, VmapMode].
```

**and I find the main problem is that self.forward_ret_dict['center_origin_cls_labels'] is a zero matrix, leading to center_origin_loss_box is a empty list. Could you please tell me how to solve this bug?**

<img width="605" alt="image" src="https://user-images.githubusercontent.com/23117298/161576229-1980d2ff-6fc8-4141-9b50-9aadfd0fbbad.png">

**Comment by yifanzhang713** [comment]
Hi @dk-liang, Thanks for your interest in our work. 

I have no idea the specifics of your own dataset and can only give some suggestions here. From the results of all-zero tensor, the reason should be that the seed points used for voting failded to fall in or around their relevant bounding boxes, you may as well try by expand the range of seed points used for voting by adjusting the **MODEL.POINT_HEAD.TARGET_CONFIG.ASSIGN_METHOD.EXTRA_WIDTH** or just set the **MODEL.POINT_HEAD.TARGET_CONFIG.ASSIGN_METHOD.ASSIGN_TYPE** with **'centers'**. 

Hope these can help you！

**Comment by darrenjkt** [comment]
Hi I managed to successfully train a model on waymo data using your provided config with x,y,z,intensity,elongation however, when I tried with x,y,z only (and all other configurations the same), I was met with this error that is similar to the above. Could I get some help on this bug?

```
center_origin_loss_box = torch.cat(center_origin_loss_box, dim=-1).mean()
RuntimeError: There were no tensor arguments to this function (e.g., you passed an empty list of Tensors)
```

For context, I'm training with x,y,z only to try and adapt this model to other datasets.
